source/js/components/petition/petition.scss

```diff
@@ -30,7 +30,7 @@
       transition: margin-bottom 0.3s ease-in-out, visibility 0s ease-in-out 0.3s;
     }
 
-    @media (min-width: $bp-md) {
+    @media (min-width: $bp-lg) {
```

Comment:
How do I get the sticky button to show up?

Reply:
just make the page narrow enough.

Reply:
if you load the page while wide, and then make it narrow, you'll need to refresh.

This is because the button goes away if the form is visible, and if the page is wide on load, the button hides.

Reply:
I think it wasn't showing up because there was no body content. I've added some and now it seems works again https://foundation-mofostaging-pr-1555.herokuapp.com/campaigns/single-page/ (note: just resizing the window doesn't work, it only shows up if you load the page at a narrow width).